main.py

```diff
@@ -670,22 +590,62 @@ def sizechanged(e):
     manager_button.configure(width=int(app.winfo_width() / 12))
 
 
-# THEMED IMAGES
+def make_windows_blur(window):
+    from ctypes import windll
+
+    from BlurWindow.blurWindow import GlobalBlur
+
+    GlobalBlur(
+        windll.user32.GetParent(window.winfo_id()),
+        Acrylic=True,
+        Dark=(theme == "Dark"),
+        hexColor=bg_color,
+    )
+
+
+# LOAD IMAGES

```

Comment:
Better to have the image definitions here, than creating them every time the `createSettingsWindow` function runs.